src/components/Book.jsx

```diff
@@ -15,9 +15,9 @@ const Book = ({ info }) => {
       <div className="book-item">
         <div className="left-card">
           <div className="information">
+            <p>{info.category}</p>
             <h4>{info.title}</h4>
             <small>{info.author}</small>
```

Comment:
- [ ] Good job @mahabubx7, however, Kindly make sure to stick to the design as it is required  👍 

> | | Zaplin template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-05-09 164632](https://github.com/mahabubx7/bookstore/assets/57408419/766882a0-c89c-49a9-94ec-a334ec0b2c96)| |![ss](https://github.com/mahabubx7/bookstore/assets/57408419/e7787b4a-3dec-4602-850b-a648f2b180d7)| |

For documentation, you may check [this link](https://www.npmjs.com/package/react-circular-progressbar) 😃